src/vim9.h

```diff
@@ -151,6 +151,9 @@ typedef enum {
     ISN_COMPAREFUNC,
     ISN_COMPAREANY,
 
+    // bitwise left/right shift operation
```

Comment:
This should be another case of `ISN_OPNR` instead of having its own opcode.

Reply:
I have updated the PR to use ISN_OPNR instead of using a new opcode.